src/docs/pages/component-docs/components/user-menu/user-menu-doc.component.ts

```diff
@@ -110,14 +112,14 @@ import { UserMenuPlaygroundKnobs } from './examples/playground.component';
                         ></app-view-code-button>
                         <app-copy-code-button [code]="PLACEMENT"></app-copy-code-button>
                     </div>
-                </div>
+                </div> -->
 
                 <div class="example-section">
-                    <div class="example-heading">Responsive User Menu</div>
+                    <div class="example-heading">Using a Bottom Sheet</div>
                     <div class="example-description">
-                        The <code>&lt;blui-user-menu&gt;</code> by default renders (or transitions if already opened) as
-                        a bottomsheet if the window width is less than 600px. The <code>useBottomSheetAt</code> property
-                        is used to adjust the screen width where this transition occurs.
+                        On larger screens the User Menu opens as a dropdown, but on smaller screens it opens as a
+                        bottomsheet. You can customize the point where this transition occurs via the
+                        <code>useBottomSheetAt</code> input (default of 600px).
```

Comment:
```suggestion
                        <code>useBottomSheetAt</code> input (default to 600px).
```

Reply:
Is this "100_00" some syntax I never heard of

![image](https://user-images.githubusercontent.com/8997218/193908693-be133500-802d-4e7d-8960-1bad11cd0479.png)

Reply:
In newer versions of javascript the underscore can be added to add separation between groups of 0s.  It's something that'll start to appear more often.